modules/core/src/test/scala/org/tessellation/http/routes/TrustRoutesSuite.scala

```diff
@@ -63,4 +85,67 @@ object TrustRoutesSuite extends HttpSuite {
       }
       .unsafeRunSync()
   }
+
+  test("GET trust latest succeeds") {
+    val req = GET(uri"/trust/latest")
+
+    val gen = for {
+      trustInfo <- Gen.mapOfN(4, genPeerTrustInfo)
+    } yield TrustMap.empty.copy(trust = trustInfo)
```

Comment:
```suggestion
    val gen = for {
      numTrusts <- Gen.chooseNum(0, 10)
      trustInfo <- Gen.mapOfN(numTrusts, genPeerTrustInfo)
    } yield TrustMap(trust = trustInfo, PublicTrustMap.empty)
```

Since this generator is used in all three tests I suggest making it a function

Reply:
Extracted to `genTrustMap`.